### DISCUSSION TITLE
README.md math formula error

### DISCUSSION THREAD

**Discussion by cloudy-sfu** [community/community#20154] [Other Feature Feedback, Questions, & Ideas] [answered]
Currently, there is an error of displaying inline math formula in README.md markdown.

The markdown code:

```
Denote $\bar x$ as the mean of each variable and $\sigma(x)$ as the standard deviation of each variable. 
For all variables, assign outliers which is smaller than $\bar x - 3 \sigma(x)$ to $\bar x - 3 \sigma(x)$. 
Assign outliers which is larger than $\bar x + 3 \sigma(x)$ to $\bar x + 3 \sigma(x)$.
```

Preview in other markdown software:

![image](https://user-images.githubusercontent.com/41314224/177773047-ab33797a-dc27-4cd1-bc1a-aefea715a1bb.png)

Preview in GitHub readme:

![image](https://user-images.githubusercontent.com/41314224/177773153-e25e3356-8238-4394-8bac-bcbd47a2ff70.png)

Hope this problem can be resolved soon.

**Comment by Gustavo2022003** [comment]
I think you can find your answer in this discussion: https://gist.github.com/a-rodin/fef3f543412d6e1ec5b6cf55bf197d7b

**Comment by nschloe** [comment, marked as answer]
This appears to have been fixed by now:

> Denote $\bar x$ as the mean of each variable and $\sigma(x)$ as the standard deviation of each variable. 
> For all variables, assign outliers which is smaller than $\bar x - 3 \sigma(x)$ to $\bar x - 3 \sigma(x)$. 
> Assign outliers which is larger than $\bar x + 3 \sigma(x)$ to $\bar x + 3 \sigma(x)$.
